tsconfig.eslint.json

```diff
@@ -1,5 +1,9 @@
 {
 	"extends": "./tsconfig.json",
+	"compilerOptions": {
+		"lib": [],
+		"types": ["vitest/importMeta", "@typescript/lib-esnext", "@typescript/lib-dom"],
+	},
```

Comment:
use `lib` from `node_modules/@typescript/lib-*` instead of `node_modules/typescript/lib/*`